### PR TITLE
User can view all listed articles

### PR DESCRIPTION
**Pull request by KyunginNa**
[Visitor can view all listed articles MOBILE](https://www.pivotaltracker.com/story/show/176444798)
## User Story
```
As a Visitor,
In order to see the available articles, 
I would like to be able to see a list of articles in the mobile app.
```
## Changes proposed in this pull request:
* Creating HomeScreen component
* Creating ArticlesService module
* Creating DisplayArticlesList component
* Adding FETCH_ARTICLE_INDEX action in rootReducer

## What I have learned working on this feature:
* How to work with React Navigation
* How to do styling in react-native

## Packages/Gems added
* axios
* react-native-gesture-handler
* react-native-reanimated
* react-native-safe-area-context
* react-native-screens
* react-native-web

## Screenshots (Client)
Screenshot 2021-01-14 at 15 08 11

### CODE REVIEW

File: components/HomeScreen.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from "react";
+import { StyleSheet, View, FlatList } from "react-native";
+import { useSelector } from "react-redux";
+import ArticleServices from "../modules/ArticleServices";
+import { StatusBar } from 'expo-status-bar'
+import DisplayArticlesList from './DisplayArticlesList'
+
+const styles = StyleSheet.create({
```
Comment (Ehugo2000) [inline components/HomeScreen.jsx]: For consistency try to have our `const styles` in the end (or top) on the component.

**Review by Ehugo2000 — COMMENTED** [review]
one comment! nice work!

File: components/HomeScreen.jsx (hunk 1 above)
Comment (ViktorHek) [reply components/HomeScreen.jsx]: done! :)

**Review by tochman — APPROVED** [review]
Looks good to me. Good work

**Pull request merged**